MoreLinq.Test/AtMostTest.cs

```diff
@@ -0,0 +1,176 @@
+#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2008 Jonathan Skeet. All rights reserved.
```

Comment:
The copyright notices should correspond to the author aka you. See 57295fff782ddc22d57c3883ed70dd77bd06e338.